src/components/application_manager/src/app_service_manager.cc

```diff
@@ -68,6 +68,37 @@ smart_objects::SmartObject AppServiceManager::PublishAppService(
   service_record[strings::service_active] = true;
   published_services_.insert(std::pair<std::string, smart_objects::SmartObject>(
       service_id, service_record));
+
+  smart_objects::SmartObjectSPtr notification =
```

Comment:
Either in the appServiceManager, or the application app service extension, should have a service record object that stores of the service records for all published app services. This might need to be a separate task on its own PR

Reply:
Resolved. Made changes to notification to include all published services, not just the updated one